src/backend/EdFi.DataManagementService.Backend.Postgresql/Operation/GetDocumentById.cs

```diff
@@ -47,13 +49,17 @@ NpgsqlTransaction transaction
                 getRequest.TraceId
             );
 
-            if (edfiDoc == null)
+            if (document == null)
             {
                 return new GetResult.GetFailureNotExists();
             }
 
-            // TODO: Documents table needs a last modified datetime
-            return new GetResult.GetSuccess(getRequest.DocumentUuid, edfiDoc, DateTime.Now);
+            return new GetResult.GetSuccess(
+                getRequest.DocumentUuid,
+                document.EdfiDoc.Deserialize<JsonNode>()!,
+                document.LastModifiedAt!.Value,
```

Comment:
Don't use `!` here. You can't guarantee those values won't be null. Use a default value instead. Or, better yet, use default values in `FindDocumentEdfiDocByDocumentUuid().`